### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Misha20850**
- [DEMO LINK](https://Misha20850.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Misha20850.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+.wrapper {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
```
Comment (etojeDenys) [inline src/style.css:23]:
```suggestion

```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav {
+  margin-right: 50px;
+}
```
Comment (etojeDenys) [inline src/style.css:58]:
```suggestion

```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background: #e5e5e5;
+}
+
```
Comment (etojeDenys) [inline src/style.css:13]:
```suggestion
.header {
 padding: 0 50px;
}
```

File: src/index.html [hunk 4]
```diff
@@ -15,8 +15,108 @@
...
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
```
Comment (etojeDenys) [inline src/index.html:33]: this wrapper looks redundant

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,78 @@
...
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (etojeDenys) [inline src/style.css:61]: better to move this color to some var and reuse it everywhere

**Review by etojeDenys — CHANGES_REQUESTED** [review]
few minor comments

File: src/style.css (hunk 2 above)
Comment (Misha20850) [reply src/style.css:57]: I don't understand where the mistake is

File: src/style.css (hunk 5 above)
Comment (Misha20850) [reply src/style.css:60]: I didn't Sass the exam

**Pull request closed without merging**